shared/common-adapters/text.desktop.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import React, {Component} from 'react'
+// TODO remove this from this component, hook it in externally so we don't have these types of dependencies in storybook
 import openURL from '../util/open-url'
```

Comment:
i chose text cause it has minimal external dependencies is and mostly clean but this pulls in some electron specific code, we should likely remove that